pilot/pkg/bootstrap/istio_ca.go

```diff
@@ -333,7 +333,7 @@ func (s *Server) handleCACertsFileWatch() {
 
 		case event, ok := <-s.cacertsWatcher.Events:
 			if !ok {
-				log.Info("Failed to catch events on cacerts files")
+				log.Error("Failed to catch events on cacerts files")
```

Comment:
why not return here and also when `Errors` channel closed.

Reply:
It doesn't work whenever these channel closed

Reply:
Updated, thanks!